src/main/java/com/epam/test_generator/services/jenkins/JenkinsJobServiceImpl.java

```diff
@@ -43,16 +41,17 @@ public JenkinsJobServiceImpl(
             Map<String, Job> jobs = jenkinsServerFactory.getJenkinsServer().getJobs();
 
             List<CommonJenkinsJobResponse> response = jobs.values().stream()
-                .map(this::transform).collect(Collectors.toList());
+                .map(job -> transform(job)).collect(Collectors.toList());
             return response;
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            e.printStackTrace();
```

Comment:
Kindly replace printing to cmd channel with printing into log file.
Use logger

Reply:
it's appear here from merge with develop, it's another task